packages/theme/src/cli/flags.ts

```diff
@@ -13,4 +13,17 @@ export const themeFlags = {
     env: 'SHOPIFY_FLAG_PATH',
     default: '.',
   }),
+  password: Flags.string({
+    hidden: false,
+    description: 'Password generated from the Theme Access app.',
+    env: 'SHOPIFY_CLI_THEME_TOKEN',
```

Comment:
I'd be consistent with the flags convention because with presets flags will become the primitive for configuring workflows. `SHOPIFY_CLI_PARTNERS_TOKEN` should indeed have been a flag with its counterpart environment variable.